src/libs/fileDownload/getAttachmentDetails.js

```diff
@@ -0,0 +1,39 @@
+import CONST from '../../CONST';
+import Config from '../../CONFIG';
+
+/**
+ * Extract the thumbnail URL, source URL and the original filename from the HTML.
+ * @param {String} html
+ * @returns {Object}
+ */
+const PREVIEW_SOURCE_REGEX = new RegExp(`${CONST.ATTACHMENT_PREVIEW_ATTRIBUTE}*=*"(.+?)"`, 'i');
+const SOURCE_REGEX = new RegExp(`${CONST.ATTACHMENT_SOURCE_ATTRIBUTE}*=*"(.+?)"`, 'i');
+const ORIGINAL_FILENAME_REGEX = new RegExp(`${CONST.ATTACHMENT_ORIGINAL_FILENAME_ATTRIBUTE}*=*"(.+?)"`, 'i');
+
+export default function getAttachmentName(html) {
+    if (!html) {
+        return {
+            previewSourceURL: null,
+            sourceURL: null,
+            originalFileName: null,
+        };
+    }
+    let previewSourceURL = html.match(PREVIEW_SOURCE_REGEX)[1];
```

Comment:
Let's change this to `const` and chain the `.replaces()` here and for `sourceURL`.